### DISCUSSION TITLE
dockerfile examples

### DISCUSSION THREAD

**Discussion by Manubi** [Dokploy/dokploy#71] [Ideas]
Hi,
Would be really nice to have dockerfile example for the most common usecases.

**Comment by Siumauricio** [comment]
Good idea!, in the future we can create a repository with a list of optimized dockerfile for most common usecases like nextjs, nestjs, react app, astro and so on..

**Comment by Mr1Blaze** [comment]
Maybe will be a good way to create service templates or plugins. Will be much easier
